appveyor.yml

```diff
@@ -1,17 +1,24 @@
-build: off
+# version: 1.0.{build}-{branch}
 
-# ruby versions under test
-platform:
-  - Ruby23-x64
-  - Ruby21-x64
+environment:
+  matrix:
+    - RUBY_VERSION: 23
```

Comment:
It looks like this changes from running 64bit Ruby to running 32 bit Ruby ( https://www.appveyor.com/docs/build-environment/#ruby )
Does that matter?